adapters/flipp/flipp.go

```diff
@@ -20,16 +20,20 @@ import (
 )
 
 const (
-	bannerType      = "banner"
-	inlineDivName   = "inline"
-	flippBidder     = "flipp"
-	defaultCurrency = "USD"
+	bannerType                  = "banner"
+	inlineDivName               = "inline"
+	flippBidder                 = "flipp"
+	defaultCurrency             = "USD"
+	defaultStandardHeight int64 = 2400
+	defaultCompactHeight  int64 = 600
 )
 
 var (
-	count    int64 = 1
-	adTypes        = []int64{4309, 641}
-	dtxTypes       = []int64{5061}
+	count          int64 = 1
+	adTypes              = []int64{4309, 641}
+	dtxTypes             = []int64{5061}
+	flippExtParams openrtb_ext.ImpExtFlipp
+	key            string
```

Comment:
you can just initialize the key here to be `standardHeight`

Reply:
initializing the key there to be standardHeight causes some errors. I think it's because the key will never be modified again in buildBid to standardHeight from compactHeight if it's needed @hasan-kanjee